LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2019 Kristoffer Carlsson <kristoffer.carlsson@juliacomputing.com>
```

Comment:
```suggestion
Copyright (c) 2015-2020 Kristoffer Carlsson, Simon Danisch, Luca Trevisani, Julia Computing, and contributors.
```

Reply:
@ViralBShah

Reply:
Not sure who is able to accept the suggested change. Perhaps either @KristofferC or @SimonDanisch